lib/dry/operation/class_context/prepend_manager.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require "dry/operation/errors"
+
+module Dry
+  class Operation
+    module ClassContext
+      # @api private
+      class PrependManager
+        def initialize(klass:, methods_to_prepend:, prepended_methods: [])
+          @klass = klass
+          @methods_to_prepend = methods_to_prepend
+          @prepended_methods = prepended_methods
+        end
+
+        def register(*methods)
+          check_pristine
```

Comment:
A name I like to use for methods like this is "ensure", so in this case `ensure_pristine`. I think it more clearly implies that _something bad’s gonna happen_ in the case of the object not being pristine.

Renaming it like so would also provide greater consistency across our dry-rb code: https://github.com/search?q=org%3Adry-rb+%22def+ensure%22&type=code

Reply:
It makes sense. Changed!